packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js

```diff
@@ -296,15 +297,27 @@ export default class QuanticTabBar extends LightningElement {
    * Toggles the combobox.
    * @returns {void}
    */
-  toggleCombobox() {
+  toggleCombobox(event) {
```

Comment:
It is out of the scope of this PR, but I wonder why the term "Combobox" is used here. A "Combobox" is a combination of an input box and a dropdown. "Dropdown" should have been used instead.

Reply:
Correct! There is no reason to choose the name "Combobox" over "Dropdown", I will update the naming, it will be more logical.